UI/Headless/Fixture.cpp

```diff
@@ -44,26 +45,41 @@ class HttpEchoServerFixture final : public Fixture {
     Optional<Core::Process> m_process;
 };
 
-ErrorOr<void> HttpEchoServerFixture::setup()
+ErrorOr<void> HttpEchoServerFixture::setup(WebView::WebContentOptions& web_content_options)
 {
-    auto script_path = LexicalPath::join(s_fixtures_path, m_script_path);
+    auto const script_path = LexicalPath::join(s_fixtures_path, m_script_path);
+    auto const arguments = Vector { script_path.string(), "--directory", Ladybird::Application::the().test_root_path };
 
     // FIXME: Pick a more reasonable log path that is more observable
-    auto log_path = LexicalPath::join(Core::StandardPaths::tempfile_directory(), "http-test-server.log"sv).string();
+    auto const log_path = LexicalPath::join(Core::StandardPaths::tempfile_directory(), "http-test-server.log"sv).string();
+
+    auto stdout_fds = TRY(Core::System::pipe2(0));
 
-    auto arguments = Vector { script_path.string(), "start", "--directory", Ladybird::Application::the().test_root_path };
-    auto process_options = Core::ProcessSpawnOptions {
+    auto const process_options = Core::ProcessSpawnOptions {
         .executable = Ladybird::Application::the().python_executable_path,
         .search_for_executable_in_path = true,
         .arguments = arguments,
         .file_actions = {
-            Core::FileAction::OpenFile { ByteString::formatted("{}.stdout", log_path), Core::File::OpenMode::Write, STDOUT_FILENO },
-            Core::FileAction::OpenFile { ByteString::formatted("{}.stderr", log_path), Core::File::OpenMode::Write, STDERR_FILENO },
+            Core::FileAction::DupFd { stdout_fds[1], STDOUT_FILENO },
```

Comment:
This construction means we have no visibility into whether any errors occurred from the script itself, which might hurt future extensions. With the previous one, we at least got a log file we could hunt for.

If we don't care about any stdout after reading the port number, we should probably keep the stderr log file as a breadcrumb for our future selves

Reply:
Re-added the stderr log file